scripts/codegen/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+FOLDER="./tmp/service-contracts"
+# rm -rf ./tmp
+mkdir -p ./tmp
+
+git -C ${FOLDER} pull || git clone git@github.com:cognitedata/service-contracts.git ${FOLDER}
+cd ${FOLDER}
+yarn install
+chmod +x build.sh
+./build.sh
+cd ../..
+
+yarn install
+node scripts/codegen/index.js
```

Comment:
Can we split up the tasks of generating the types vs. linting and building? So one can work on this with type generation while not doing the rest